src/js/helpers/cleanData.js

```diff
@@ -94,9 +94,9 @@ export const cleanContact = contact => {
   return cleaned;
 };
 
-export const cleanLocationPageJanisUrl = janisUrl => {
-  if (!!janisUrl.length) {
-    return janisUrl[0];
+export const cleanLocationPageUrl = janisUrls => {
+  if (!!janisUrls.length) {
+    return janisUrls[0];
```

Comment:
Don't know why this worked either! But makes sense to be urls, due to the fact it's an array.

Reply:
The reason it worked before is that in prod right now the schema for ServicePageNode is
```
class ServicePageNode(DjangoObjectType):
    page_type = graphene.String()
    janis_url = graphene.List(graphene.String)
    owner = graphene.Field(OwnerNode)
```
so it is janis_url on the relatedService. your joplin refactoring fixes it

Reply:
Oh interesting. Thanks for resolving that mystery for me! I couldn't figure out how it was working before.